code/modules/antagonists/eldritch_cult/knowledge.dm/flesh_lore.dm

```diff
@@ -0,0 +1,257 @@
+		H.ghostize(0)
+		H.key = C.key
+	if(!H.mind || !H.client)
+		to_chat(user, "<span class='warning'>There is no soul connected to this body...</span>")
+		return
+	if(!check_ghouls() || HAS_TRAIT(H,TRAIT_GHOUL))
+		return
+
+	ADD_TRAIT(H,TRAIT_MUTE,MAGIC_TRAIT)
+	ADD_TRAIT(H,TRAIT_GHOUL,MAGIC_TRAIT)
+	H.revive(full_heal = TRUE, admin_revive = TRUE)
+	H.setMaxHealth(50)
+	H.health = 50 // Voiceless dead are much tougher than ghouls
+	H.fully_replace_character_name(H.real_name,"Voiceless [H.real_name]")
+	to_chat(H, "<span class='userdanger'>You have been revived by </span><B>[user.real_name]!</B>")
+	to_chat(H, "<span class='userdanger'>[user.p_theyre(TRUE)] your master now, assist [user.p_them()] even if it costs you your new life!</span>")
+	atoms -= H
+	ghouls += H
+
+/datum/eldritch_knowledge/flesh_ghoul/proc/check_ghouls()
+
+	listclearnulls(ghouls)
+
+	for(var/mob/living/carbon/human/ghoul in ghouls)
+		if(ghoul.stat == DEAD)
+			ghouls -= ghoul
+			current_amt--
+
+	if(current_amt >= max_amt)
+		return FALSE
+	return TRUE
+
+/datum/eldritch_knowledge/flesh_grasp
+	name = "Grasp of Flesh"
+	gain_text = "My new found desire, it drove me to do great things! The Priest said."
+	desc = "Empowers your mansus grasp to be able to create a single ghoul out of a dead person. Ghouls have only 25 HP and look like husks."
+	cost = 1
+	next_knowledge = list(/datum/eldritch_knowledge/flesh_ghoul)
+	var/ghoul_amt = 1
+	var/list/spooky_scaries
+	route = "Flesh"
+
+/datum/eldritch_knowledge/flesh_grasp/mansus_grasp_act(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!ishumanbasic(target))
+		return
+	var/mob/living/carbon/human/H = target
+	var/datum/status_effect/eldritch/E = H.has_status_effect(/datum/status_effect/eldritch/rust) || H.has_status_effect(/datum/status_effect/eldritch/ash) || H.has_status_effect(/datum/status_effect/eldritch/flesh)
+	if(E)
+		E.on_effect()
+		H.bleed_rate = min(H.bleed_rate + 4, 8)
+	if(H.stat != DEAD)
+		to_chat(user, "<span class='warning'>This spell can only affect the dead!</span>")
+		return
+
+	for(var/mob/dead/observer/ghost in GLOB.dead_mob_list) //excludes new players
+		if(ghost.mind && ghost.mind.current == H && ghost.client)  //the dead mobs list can contain clientless mobs
+			ghost.reenter_corpse()
+			break
+	//Remember to un comment it when it gets TMed
+	//if(!H.mind || !H.client)
+	//	to_chat(user, "<span class='warning'>There is no soul connected to this body...</span>")
+	//	return
+
+	check_ghouls()
+
+	if(LAZYLEN(spooky_scaries) >= ghoul_amt)
+		to_chat(user, "<span class='warning'>Your patron cannot support more ghouls on this plane!</span>")
+		return
+
+	LAZYADD(spooky_scaries,H)
+
+	ADD_TRAIT(H,TRAIT_GHOUL,MAGIC_TRAIT)
+	H.revive(full_heal = TRUE, admin_revive = TRUE)
+	H.setMaxHealth(25)
+	H.health = 25
+
+	H.fully_replace_character_name(H.real_name,"Ghouled [H.real_name]")
```

Comment:
send this to your friends to totally ghoul them!

Reply:
what?

Reply:
it's a joke, never mind